tests/e2e-leg-9/online-upgrade-pods-pending/setup-vdb/base/setup-vdb.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
 spec:
   image: kustomize-vertica-image
   dbName: repUP
-  initPolicy: Create
+  initPolicy: CreateSkipPackageInstall
```

Comment:
Why?

Reply:
As the test online-upgrade-pods-pending stops at the steps of wait-for-scs-mimic, the package installation (at the step of a new Vertica version is installed) is not covered in the test. So we can either
1. use CreateSkipPackageInstall to skip 
2. add the upgrade steps back to cover the package installation